ruby.md

```diff
@@ -1951,25 +1925,6 @@ no parameters.
   end
   ```
 
-* <a name="no-non-nil-checks"></a>
-  Don't do explicit non-`nil` checks unless you're dealing with boolean
```

Comment:
I think there are times when doing an explicit `nil` check is intention-revealing, so I'm happy to exclude this from the guide.